### DISCUSSION TITLE
Rustling checking every exercise file upon verifying answer.

### DISCUSSION THREAD

**Discussion by Kaonashi-R** [rust-lang/rustlings#2112] [Q&A] [answered]
Hi, I'm new to rust, and I figured I'd use rustling to learn it. I followed the installation guide on windows, but whenever I check the answer for one single exercise, rustling is checking the answers for every single one. (I'm on VS code and I see each exercise being debugged). I went for VS code instead of an IDE like rustrover for performance issues, but now it's taking ages to check if I replaced one line right... 
Is this intended ? If not, how can I fix it ?

**Comment by mo8it** [comment, marked as answer]
Duplicate of https://github.com/rust-lang/rustlings/issues/2071

Look at that issue for further information. We are working on improving it :)
